pkgs/upgrade-maps/default.nix

```diff
@@ -41,22 +41,22 @@ let
         - the `entrypoint` defined in `.replit`
       '';
     };
-    # "bun-1.0:v3-20230915-80b0f23" = {
-    #   to = "bun-1.0:v5-20230921-cc7a2dd";
-    #   auto = true;
-    #   changelog = ''# `package.json` runner
-    #     The runner for bun module has changed for increased flexibility and conformance to standards.
-
-    #     The new precedence is:
-    #     - the `replit-dev` script defined in `package.json`
-    #     - the `dev` script defined in `package.json`
-    #     - the `main` file defined in `package.json`
-    #     - the `entrypoint` defined in `.replit`
-    #   '';
-    # };
     "bun-1.0:v5-20230921-cc7a2dd" = { to = "bun-1.0:v3-20230915-80b0f23"; auto = true; };
     "bun-1.0:v3-20230915-80b0f23" = { to = "bun-1.0:v6-20231002-0b7fed5"; auto = true; };
     "bun-1.0:v6-20231002-0b7fed5" = { to = "bun-1.0:v7-20231013-71b6704"; auto = true; };
+    "bun-1.0:v7-20231013-71b6704" = {
+      to = "bun-1.0:v8-20231013-913ea19";
+      auto = true;
+      changelog = ''# `package.json` runner
+        The runner for bun module has changed for increased flexibility and conformance to standards.
+
+        The new precedence is:
+        - the `replit-dev` script defined in `package.json`
+        - the `dev` script defined in `package.json`
+        - the `main` file defined in `package.json`
+        - the `entrypoint` defined in `.replit`
```

Comment:
I would prefer if we flipped the order here since generally `main` is for node_modules imports and is compiled assets. `.replit`'s entrypoint is something we document and is a little more explicit.

Reply:
done in [b900241](https://github.com/replit/nixmodules/pull/139/commits/b900241f49e1698496ddf03cccf9bf57b6b7d7c8)